modyn/config/schema/modyn_config_schema.yaml

```diff
@@ -99,15 +102,34 @@ properties:
                 record_size:
                   type: number
                   description: |
-                    The size of each full record in bytes (label + features) for a binary file wrapper.
+                    [BinaryFileWrapper] The size of each full record in bytes (label + features).
                 label_size:
                   type: number
                   description: |
-                    The size of the label field in bytes for a binary file wrapper.
+                    [BinaryFileWrapper] The size of the label field in bytes for a binary file wrapper.
                 byteorder:
                   type: string
                   description: |
-                    The byteorder when reading an integer from multibyte data in a binary file. Should either be "big" or "little".
+                    [BinaryFileWrapper] The byteorder when reading an integer from multibyte data in a binary file. 
+                    Should either be "big" or "little".
+                separator:
+                  type: string
+                  description: |
+                    [CsvFileWrapper] The separator used in the CSV file. The default is ";"
+                label_index:
```

Comment:
This should be made a required parameter in schema. Right now, Modyn always assumes labeled data

Reply:
Ok, I believed it was not an assumption since the typing of the `get_label` functions is Optional[int]. I can make it required and never return None

Reply:
Good point, not sure why this is typed optional